exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -132,7 +132,7 @@
                 <#list model.getOptionList() as option>
                   <tr>
                     <td>${option.getName()}</td>
-                    <td>${option.getValue()?c}</td>
+                    <td>${option.getValue()?string}</td>
```

Comment:
According to the [documentation](), ?string works only for numbers and formats them for "human display". ?c also works for numbers (and formats them as "C" language constants.)

The challenge is that this template line works for all data types. So, the suggestion of using toString( ) is good. But, since toString( ) may be used for debugging, perhaps add a toDisplayString( ) that will format the value as we want it to appear in the Web UI.

Another issue: are we sure that the value of option is always non-null at this point?